engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/EngineCallCacheAccesser.scala

```diff
@@ -0,0 +1,47 @@
+package cromwell.engine.workflow.lifecycle.execution.callcaching
+
+import cromwell.backend.BackendJobExecutionActor.SucceededResponse
+import cromwell.core.{JobOutputs, WorkflowId}
+import cromwell.database.sql.CallCachingStore
+import cromwell.database.sql.tables.{CallCachingMetainfoEntry, CallCachingResultSimpletonEntry}
+import cromwell.engine.workflow.lifecycle.execution.callcaching.EngineJobHashingActor.CallCacheHashes
+import cromwell.core.ExecutionIndex.IndexEnhancedIndex
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * Given a database-layer CallCacheStore, this accesser can access the database with engine-friendly data types.
+  */
+class EngineCallCacheAccesser(database: CallCachingStore) {
```

Comment:
I'm not a fan of the existing `SqlJobStore` type names, but I can't say I'm a fan of this one either.  😄